dm/docs/RFCS/20220124_enhance_task_manageability.md

```diff
@@ -0,0 +1,131 @@
+| create   | `dmctl source create source1.yaml`                 |                         | config-file              | create source with config file.                                     |
+| update   | `dmctl source update source1 source1.yaml`         |                         | source-name, config-file | update a source with config file.                                   |
+| delete   | `dmctl source delete --force source1`              | --force(default: false) | source-name              | delete a source with config file.                                   |
+| get      | `dmctl source get --output="source1.yaml" source1` | --output(default: ".")  | source-name              | show the source config in yaml format, also support output to file. |
+| list     | `dmctl source list`                                |                         |                          | list all source in current cluster.                                 |
+| status   | `dmctl source status source1`                      |                         | source-name              | show source detail status.                                          |
+| enable   | `dmctl source enable source1`                      |                         | source-name              | enable a disabled source.                                           |
+| disable  | `dmctl source disable source1`                     |                         | source-name              | disable a source and also stop the running subtasks of this source. |
+| transfer | `dmctl source transfer source1 worker1`            |                         | source-name, worker-name | transfers a source to a free worker.                                |
+
+### dmctl commands for Relay
+
+| Command | Full Syntax Example                                                                                 | Flags                      | Arguments              | Description                                                                  |
+|---------|-----------------------------------------------------------------------------------------------------|----------------------------|------------------------|------------------------------------------------------------------------------|
+| start   | `dmctl relay start --worker-name="worker1" source1`                                                 | --worker-name(default: "") | source-name            | start relay for a source on a worker.                                        |
+| stop    | `dmctl relay stop --worker-name="worker1" source1`                                                  | --worker-name(default: "") | source-name            | stop relay for a source on a worker.                                         |
+| purge   | `dmctl relay purge --sub-dir="2ae76434-f79f-11e8-bde2-024ac130008.000001" source1 mysql-bin.000006` | --sub-dir(default: "")     | source-name, file-name | purges relay log files of the DM-worker according to the specified filename. |
```

Comment:
we have time-based purge in my memory :thinking:

Reply:
don't know why not support now https://github.com/pingcap/tiflow/blob/9858cb8e630b3020fde3bf160bab1a97d6ee988d/dm/dm/ctl/master/purge_relay.go#L106